charts/quickwit/values.yaml

```diff
@@ -178,24 +196,30 @@ metastore:
 
   # Extra env for metastore
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed